package.json

```diff
@@ -13,8 +13,8 @@
     "format-js-prettier": "prettier 'src/**/*.{json,jsx,tsx,ts}' 'tests/**/*.ts' --write",
     "format-js": "yarn format-js-eslint && yarn format-js-prettier",
     "lint-scss": "stylelint 'src/**/*.scss'",
-    "lint-js-eslint": "eslint 'src/**/*.{json,tsx,ts}' 'tests/**/*.ts' .eslintrc.cjs babel.config.js",
-    "lint-js-prettier": "prettier 'src/**/*.{json,tsx,ts}' 'tests/**/*.ts' .eslintrc.cjs babel.config.js --check",
```

Comment:
I assume the config files are removed from the script because we updated to the latest version of eslint, and also prettier does not require them?

Reply:
In order to check them here, we would need to set `allowJs` in the typescript config, which I'd like to avoid. Hence removing linting for the config files.